app/views/pages/partials/_event-card.html.erb

```diff
@@ -0,0 +1,19 @@
+
+<section id="event-card-container">
+  <div class="event-cards">
+    <ul>
+      <% @event_cards.each do |event| %>
+        <li>
+          <div class="event-cards-content">
+            <h3><%= event.day.day %></h3>
+            <div class="yoga-info">
+              <h3><%= event.title %></h3>
+              <p><%= "#{event.day.strftime("%a")} " "#{event.start_time.strftime("%I:%M")} - #{event.end_time.strftime("%I:%M")}" %></p>
+            </div>
+              <i class="fa-solid fa-circle-arrow-right"></i>
```

Comment:
indentation: i tag with the closing tag above it should be in the same level

Reply:
@haqcked when you spot mistakes like this one don't hesitate to mark your review as `Request changes`
![image](https://github.com/wJoenn/YogaUnalome/assets/75388869/7bfc5e38-2275-4513-afed-240cdfb034fa)

Reply:
You can also change the tag of the PR to be `requires changes` instead of `waiting for review`